backend/app/modules/score_delegation/core.py

```diff
@@ -33,10 +35,15 @@ def build_score_delegation_message(primary_addr: str, secondary_addr: str) -> st
 
 
 def get_hashed_addresses(
-    payload: ScoreDelegationPayload, salt: str, salt_primary: str
+    payload: ScoreDelegationPayload, salt: str, salt_primary: str, normalize: str = True
```

Comment:
```suggestion
    payload: ScoreDelegationPayload, salt: str, salt_primary: str, normalize: bool = True
```

Reply:
I recommend `pyright` for keeping the types correct

Reply:
problem with annotations occur in other places here as well; please double check everything and in case of any unlcarities, hit us up

Reply:
done